src/EA.Weee.Web/Views/Shared/_ViewEvidenceNotePartial.cshtml

```diff
@@ -16,18 +16,6 @@
             @Html.Gds().DisplayFor(m => m.Status, Model.Status.ToDisplayString())
         </div>
 
-        @if (Model.HasReturnedDate)
```

Comment:
can see the change you have made, if (Model.HasReason.... but we need to put it below the status.

![image](https://user-images.githubusercontent.com/43237115/166873209-33d5aacd-ca00-445c-9bfd-3acbbcc65fa1.png)

![image](https://user-images.githubusercontent.com/43237115/166873153-578ba8ca-545d-4598-a448-969bb6902a91.png)